src/sftp.rs

```diff
@@ -385,6 +385,8 @@ impl Sftp {
     }
 
     /// Check if the remote server supports the limits extension.
+    ///
+    /// If it returns true, then the client will follow the limits set by the server.
     pub fn support_limits(&self) -> bool {
```

Comment:
We don't need to expose this since it is only used internally.

Reply:
So should we remove `support_limits` api?

Reply:
Yes